packages/mui-base/src/utils/useSlotProps.ts

```diff
@@ -35,15 +39,12 @@ export type UseSlotPropsParameters<
 export type UseSlotPropsResult<
   ElementType extends React.ElementType,
   SlotProps,
-  ExternalForwardedProps,
-  ExternalSlotProps,
   AdditionalProps,
   OwnerState,
 > = AppendOwnerStateReturnType<
   ElementType,
-  Omit<SlotProps & ExternalSlotProps & ExternalForwardedProps & AdditionalProps, 'ref'> & {
-    className?: string | undefined;
```

Comment:
Why was the `className` removed?

Reply:
It's a part of `MergeSlotPropsResult` so this declaration was redundant.